24 - Sticky Nav/style-START.css

```diff
@@ -28,7 +28,7 @@ body {
 header {
   text-align: center;
   height: 50vh;
-  background: url(http://wes.io/iEgP/wow-so-deep.jpg) bottom center no-repeat;
+  background: url(https://source.unsplash.com/GKN6rpDr0EI/960x640) bottom center no-repeat;
```

Comment:
background: url(https://f.v1.n0.cdn.getcloudapp.com/items/331o2t1L3t1y160m3N2c/Image%202016-11-24%20at%202.45.35%20PM.jpg) bottom center no-repeat;

Reply:
Please Check that!